.circleci/config.yml

```diff
@@ -30,3 +30,27 @@ jobs:
             - store_artifacts:
                 path: doc/_build/html/
                 destination: html
+            - deploy:
+                name: github pages deployment
+                # first clone the mne-tools/mne_bids.github.io repo because it may ask
+                # for password and we don't want to delay this long build in
+                # the middle of it
+                # --no-checkout just fetches the root folder without content
+                # --depth 1 is a speed optimization since we don't need the
+                # history prior to the last commit
+                # -b gh-pages fetches only the branch for the gh-pages
+                command: |
+                  git config --global user.email "circle@mne.com";
+                  git config --global user.name "Circle Ci";
+                  cd doc
+                  rm -rf _build/doctrees _build/mne_bids.github.io
+                  git clone -b gh-pages --single-branch --no-checkout --depth 1 https://github.com/mne-tools/mne-bids _build/mne_bids.github.io
+                  touch _build/mne_bids.github.io/.nojekyll
+                  make html
+                  cd _build/ && \
+                  cp -r html/* mne_bids.github.io && \
+                  cd mne_bids.github.io && \
+                  git add * && \
+                  git add .nojekyll && \
```

Comment:
is line 54 really necessary? ... isn't .nojekyll already added via line 53?

Reply:
Hmm ... not sure. Try and see